### ISSUE TITLE
Platform Website modifications

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#56317]
## User Story
As a VFS team member, I want to know the current details of the Privacy, Security, Infrastructure Readiness Review so that I can be sure my team is prepared for it.

Assignee: @briandeconinck 
Peer Reviewer: @CherylEvans 

## Description
With the change in process for the PSIRR, the guidance available to VFS teams on Platform Website also needs to be updated. This ticket will include making all of the Platform Website changes. This ticket will also include removing language "recommend" when referring to Collab Cycle touchpoints.

## Impacted Artifacts
[Collaboration Cycle Overview](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/)
[Touchpoints](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/touchpoints)
[Collaboration cycle kickoff](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/collaboration-cycle-kickoff)
[Resources](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/resources)
[Collaboration Cycle FAQ](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/faqs)
[Privacy, security, infrastructure readiness review](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/privacy-security-infrastructure-readiness-review)
All touchpoint pages

## Tasks
Note: All changes to the pages below should be made as drafts and saved in the same folder created in #56316. No pages should be published until Roll out.
- [All documentation drafts](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2607087617/52419+-+PSIRR+Improvements+Epic)

**Changes to [Collaboration Cycle Overview](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/):**
- [x] Update visual flow image to move PSIRR under Define
  - Note: This visual is just a screenshot of a slide within the General Orientation deck. Please don't update the General Orientation deck permanently... that will come during Roll Out
- [x] Move Privacy, Security, Infrastructure Readiness Review to The "Define" Phase section, under Design Intent

**Collaboration cycle kickoff page:**
- [x] Remove the word "recommended" under "Are you ready to engage with the Collaboration Cycle?"

**Touchpoints:**
- [ ] ~Move the PSIRR touchpoint page so that it is beneath Design Intent under Touchpoints.~
   - **NOTE:** This will have to be done in the Collab Cycle space in Confluence rather than in our documentation drafts. Commenting on #56318 to make sure it makes it into that task list once it's built out. 
- [ ] ~Verify that the Touchpoints page reflects the same order~
   - See above!
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/285bea33-75b3-4e4a-ac1b-c27caa4742a4)
![image.png](https://images.zenhubusercontent.com/5f62315e822e9b2aa88b1f1c/8a8901b3-1be3-44a5-995d-6ef2b65307ef)
- [x] Remove the word "recommended" from the intro paragraph
- [x] Reorder the columns of the Platform practice area table to have PSIRR directly to the right of Design Intent
- [x] In the Platform practice area table, remove the checkmark from QA under Midpoint Review.
- [x] In the Platform practice area table, remove the checkmark from Content under Design Intent
Note: do not make any changes to Sitewide Content & IA Intake Request re: accessibility. Those will come at a later date.

**Resources page:**
- [x] Under "Awaiting Design Intent scheduling," remove the word "recommended"
- [x] Under Platform Collaboration Cycle Point Tracker, delete the "for those" statement (ie after 5/26/2022) and modify the paragraph to state that the tracker "...is used to track the products/features..."
- [x] Under Platform Collaboration Cycle Point Tracker, delete the entire 2nd paragraph (ie prior to 5/26/2022) as well as the "For those" statement

**Collaboration Cycle FAQ page:**
- [x] For question, "I’ve determined that our product needs a Collaboration Cycle review. How do I get the process started?" remove the word "recommended" in the answer section. State "We will add your touchpoints to the..."
- [x] For question, "Can I skip a Collaboration Cycle touchpoint that was recommended during Kickoff?" change the question to state "Can I skip a Collaboration Cycle touchpoint that was included at Kickoff?" In the answer, remove the 2nd instance of "recommended"

**Privacy, security, infrastructure readiness review page:**
Note: majority of this page is dependent on Platform Security. It may be wise to contact them with the requested information we need, share the current page, share the changes we are making below, and ask if there's additional changes they'd like to make. 
- [x] In the intro paragraph, bold the word **required.**
- [x] Move the sentence "PSIRR reviews are **not required** for static pages (or iterations/updates to static pages)." from Timing to the intro paragraph. Feel free to wordsmith so the 2 sentences work better together.
- [x] Under Timing, remove the verbiage and replace with:
   - "Request this touchpoint when you have an idea of how to solve your problem statement and have drafted designs. This touchpoint should be requested within the same timeframe as the Design Intent." (wordsmith if needed)
- [ ] Update Format per Platform Security feedback
   - Shira's note: Ideally it'll be 2 parts. One asynchronous review of the VFS teams submitted ticket in the Define phase, and then 1 synchronous meeting prior to the VFS team preparing for launch. However this should be confirmed/validated with Platform Security.
- [ ] Update Artifacts per Platform Security feedback
- [ ] Update Outcome per Platform Security feedback
- [ ] The link to the PSIRR Github ticket may change. Request updated link from Platform Security. 
   - Note: the link may need to be updated at a later date. If so, leave comment within draft page.

**All touchpoints pages:**
- [x] At the top of each touchpoint page (excluding PSIRR), there is a question, "Did we recommend a ______ in the Collaboration Cycle Request ticket? If so, follow the below guidelines to get started." Update all pages to state, "Did we include _______ in the Collaboration Cycle Request ticket? If so, follow the below guidelines to get started."
   - [x] Design Intent
   - [x] Sitewide Content and IA intake request
   - [x] Research review
   - [x] Midpoint review
   - [x] Analytics request
   - [x] Contact center review
   - [x] Staging review

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [x] Platform Website is updated to reflect the new PSIRR process
- [x] Platform Website no longer includes the language of "recommended" wrt touchpoints

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [x] Ticket as an assignee and peer reviewer

**Comment by briandeconinck** [comment]
Status update: All tasks completed except those that require syncing with the Platform Security team. Those are coming next!

**Comment by shiragoodman** [comment]
@briandeconinck is this ready for @CherylEvans to peer review?

**Comment by briandeconinck** [comment]
@shiragoodman Maybe? I guess I'm a little uncertain.

There are four unchecked tasks that depend on the Platform Security team. Should I be pinging them for answers at this point? I'm not sure what their timetable looks like for working on this.

**Comment by shiragoodman** [comment]
@briandeconinck  good point. I have a meeting with Jesse House on Monday and will get an update on their status at that time. To make the best use of our time, let's have Cheryl review what you currently have in place, with the caveat that some things may change dependent on PS feedback. There's so much here, so I don't see the harm in having her look now, and then potentially a 2nd review once we get answers from PS and implement the changes.

**Comment by briandeconinck** [comment]
Per Shira's comments here and in the standup, ready for PR @CherylEvans

**Comment by CherylEvans** [comment]
@briandeconinck Everything looks good so far. :-). I made one small recommendation on the Privacy, security, infrastructure readiness review page (first paragraph).  Otherwise, it all looks great!

**Comment by jhouse-solvd** [comment]
@shiragoodman - I spoke w/ the team yesterday, and we feel that it would be good to revisit this once we have clarified a few more internal processes and definitions. We want to avoid requesting information from VFS teams or placing an undue burden on their workflow. 

I recommend placing this work on hold for a few sprints so that we can revisit it once the Platform Security team has a few other pressing pieces of work out of the way. We can discuss in greater detail next week if you'd like, but I wanted to share my high-level take based on the information I've gathered. ☺️
